core/cli/commands.py

```diff
@@ -126,5 +131,17 @@ def do_back(self, arg):
         else:
         	self.namespace = ""
 
+    def do_harden(self, arg):
+        """Execute the hardening method"""
+
+        if not (self.module and self.namespace):
+            logger.error('Select a module/namespace')
+
+        hrd = self.modules[self.namespace][self.module]()
+        cmd = hrd.command()
+
+        logger.info(run_cmd(cmd))
```

Comment:
Is it okay for printing executed command's output as `info`? Maybe we can use just `print` since we don't check the output.